webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsDraftVisibilityMobile.vue

```diff
@@ -0,0 +1,72 @@
+<template>
```

Comment:
header should be added as mentioned with sonar check https://sonarcloud.io/project/issues?id=org.exoplatform.addons.news&open=AXzbsXeSS3KsrXT--xAR&pullRequest=329&resolved=false&types=CODE_SMELL
you can use this example https://github.com/exoplatform/news/pull/301/files#diff-ad2f73acd8125acd727519e2af2cf82a9e66b9f3b7921a85f3d3c97554d01803R1